recommender/app/controllers/Application.scala

```diff
@@ -43,7 +43,7 @@ object Application extends Controller {
 
   private val auth = new Auth()
 
-  val healthcheck = new Healthcheck(recommender)
+  val healthcheck = new MetricSender(recommender)
```

Comment:
I know you didn't change the name of this but I was very confused for a while as to how we can have both a val and a def called "healthcheck" in the same scope. It turns use the val is called healthcheck and the def is called healthCheck